src/libraries/System.IO.Hashing/src/System/IO/Hashing/XxHash3.State.cs

```diff
@@ -0,0 +1,170 @@
+// Licensed to the .NET Foundation under one or more agreements.
```

Comment:
This file seems misnamed. The other Xx.State.cs files are named such because they contain the corresponding Xx.State types. This, however, is the public XxHash3.